csharp/private/actions/assembly.bzl

```diff
@@ -78,12 +87,13 @@ def AssemblyAction(
     if debug:
         args.add("/debug+")
         args.add("/optimize-")
+        args.add("/define:TRACE;DEBUG")
     else:
+        args.add("/debug-")
         args.add("/optimize+")
+        args.add("/define:TRACE;RELEASE")
 
-        # TODO: .NET core projects use debug:portable. Investigate this, maybe move
-        #       some of this into the toolchain later.
-        args.add("/debug:pdbonly")
+    args.add("/debug:portable")
```

Comment:
This one is debatable. SDK-style (what we call old style) use pdbonly, but what we call modern-style use portable. Not all tools support the portable PDB format yet (?) hence the difference.

It seems most reasonable to just do the future thing. We could make it configurable if we have to. Obviously we'll need to put some more thought into this for D2L (but then again, we'll be 100% modern-style soon I hope). I can back this part of the change out for now if you want.